utils/host/src/logger.rs

```diff
@@ -42,6 +42,7 @@ fn build_env_filter() -> EnvFilter {
 /// - `OTLP_ENDPOINT`: OpenTelemetry endpoint (defaults to http://localhost:4317)
 /// - `OTLP_ENABLED`: Whether to enable OpenTelemetry export (defaults to false)
 /// - `RUST_LOG`: Standard Rust log level configuration
+/// - `LOG_FORMAT`: Whether to enable pretty logging (defaults to "pretty")
```

Comment:
```suggestion
/// - `LOG_FORMAT`: Output format (pretty or json, defaults to pretty)
```
nit: how about fixing like this since original comment sounds more like a boolean flag?